test/test_test.c

```diff
@@ -374,6 +374,45 @@ static int test_long_output(void)
            & TEST(0, TEST_mem_eq(r, strlen(r), s, strlen(s)));
 }
 
+static int test_long_bignum(void)
+{
+    int r;
+    BIGNUM *a = NULL, *b = NULL, *c = NULL, *d = NULL;
+    const char as[] = "1234567890123456789012345678901234567890123456789012"
+                     "1234567890123456789012345678901234567890123456789012"
```

Comment:
Jagged formatting. I mean it looks like one white space is missing here and in number of other places.